configs/segmentation/mask2former/mask2former_r50_8xb2_e50_instance.py

```diff
@@ -0,0 +1,209 @@
+_base_ = ['configs/base.py']
+
+CLASSES = [
+    'person', 'bicycle', 'car', 'motorcycle', 'airplane', 'bus', 'train',
+    'truck', 'boat', 'traffic light', 'fire hydrant', 'stop sign',
+    'parking meter', 'bench', 'bird', 'cat', 'dog', 'horse', 'sheep', 'cow',
+    'elephant', 'bear', 'zebra', 'giraffe', 'backpack', 'umbrella', 'handbag',
+    'tie', 'suitcase', 'frisbee', 'skis', 'snowboard', 'sports ball', 'kite',
+    'baseball bat', 'baseball glove', 'skateboard', 'surfboard',
+    'tennis racket', 'bottle', 'wine glass', 'cup', 'fork', 'knife', 'spoon',
+    'bowl', 'banana', 'apple', 'sandwich', 'orange', 'broccoli', 'carrot',
+    'hot dog', 'pizza', 'donut', 'cake', 'chair', 'couch', 'potted plant',
+    'bed', 'dining table', 'toilet', 'tv', 'laptop', 'mouse', 'remote',
+    'keyboard', 'cell phone', 'microwave', 'oven', 'toaster', 'sink',
+    'refrigerator', 'book', 'clock', 'vase', 'scissors', 'teddy bear',
+    'hair drier', 'toothbrush'
+]
+
+model = dict(
+    type='Mask2Former',
+    backbone=dict(
+        type='ResNet',
+        depth=50,
+        num_stages=4,
+        out_indices=(1, 2, 3, 4),
+        frozen_stages=-1,
+        norm_cfg=dict(type='BN', requires_grad=False),
+        norm_eval=True),
+    head=dict(
+        type='Mask2FormerHead',
+        pixel_decoder=dict(
+            input_stride=[4, 8, 16, 32],
+            input_channel=[256, 512, 1024, 2048],
+            transformer_dropout=0.0,
+            transformer_nheads=8,
+            transformer_dim_feedforward=1024,
+            transformer_enc_layers=6,
+            conv_dim=256,
+            mask_dim=256,
+            norm='GN',
+            transformer_in_features=[1, 2, 3],
+            common_stride=4,
+        ),
+        transformer_decoder=dict(
+            in_channels=256,
+            num_classes=80,
+            hidden_dim=256,
+            num_queries=100,
+            nheads=8,
+            dim_feedforward=2048,
+            dec_layers=9,
+            pre_norm=False,
+            mask_dim=256,
+            enforce_input_project=False,
+        ),
+        num_things_classes=80,
+        num_stuff_classes=0,
+    ),
+    train_cfg=dict(
+        class_weight=2.0,
+        mask_weight=5.0,
+        dice_weight=5.0,
+        deep_supervision=True,
+        dec_layers=10,
+        num_points=12554,
+        no_object_weight=0.1,
+        oversample_ratio=3.0,
+        importance_sample_ratio=0.75,
+    ),
+    test_cfg=dict(
+        instance_on=True,
+        max_per_image=100,
+    ),
+    pretrained='https://download.pytorch.org/models/resnet50-19c8e357.pth',
```

Comment:
@tuofeilunhifi check this, we should set  backbone.pretrained=True to load default model

Reply:
done